Makefile

```diff
@@ -64,6 +71,9 @@ SRC_TST            := $(ROOT)/test
 SRC_TSTS           := $(shell find $(SRC_TST) -name "*.ttl" | sort)
 FLG_TSTS           := $(patsubst $(SRC_TST)/%.ttl,$(DST_TEST)/%.tested,$(SRC_TSTS))
 
+# - Permission mode
+PERMISSION_MODE := 766
```

Comment:
A more restrictive permission is not feasible based on the various tests conducted, particularly because the files created in these directories are done so through Docker containers that mount volumes. It is possible that there might be ways to make the approach more secure, but this would require a more in-depth analysis beyond the scope of this pull request (PR).